analyses/copy_number_consensus_call/Snakefile

```diff
@@ -83,3 +85,22 @@ rule manta_filter:
         """ | sort -k1,1 -k2,2n > {output.manta_del} && """
         """awk '$6~/DUP/ {{if ($5 > {params.SIZE_CUTOFF}) {{print$2,$3,$4,$5,"NA","NA",$6}}}}' {input} """
         """ | sort -k1,1 -k2,2n > {output.manta_dup}"""
+
+
+rule filter_telomeres:
+    input:
+        ## Define the location of the input file and take the path/extension from the config file
+        script=os.path.join(config["scripts"], "get_rid_bad_segments.py"),
+        bad_list=os.path.join(config["scripts"], "bad_chromosomal_seg_updated_merged.txt"),
```

Comment:
As per my other suggestion, should this be renamed?

Reply:
Yes, I will make that change